test/integration/features/story_health.feature

```diff
@@ -4,24 +4,15 @@ Feature:
     project and deploy an app. Check on the app and delete the
     project. Stop and delete CRC.
 
-    Scenario Outline: Start CRC
+    @linux @darwin
+    Scenario: Start CRC
```

Comment:
Not sure about these changes from `Scenario Outline` to `Scenario`? This happens several times in this patch.

Reply:
@cfergeau if you use `Scenario Outline` you need to put `examples` fields with table like it was there before the change but since we dont' need the table so it need to be changed also.

Reply:
@cfergeau as @praveenkumar says, maybe we can delete the table, change `Scenario Outline` to `Scenario` and replace `starting CRC with default bundle and hypervisor <vm-driver> succeeds` with 
```
starting CRC with default bundle and default hypervisor succeeds
```
This will **not** use the `--vm-driver` option. I can clean up the "hypervisor" wording from the text later.

Another option is that I submit a PR tomorrow changing these things, then you rebase onto that (if you want).

Reply:
Did you look at the PR? I believe this already is doing the changes you are thinking of?

Reply:
My bad, just looked at these comments instead of the whole PR.